charts/home-assistant/2.0.0/questions.yaml

```diff
@@ -0,0 +1,532 @@
+                      schema:
+                        type: path
+                        required: true
+                        default: "/dev/ttyUSB0"
+                        editable: true
+                    - variable: hostPathEnabled
+                      label: "host Path Enabled"
+                      schema:
+                        type: boolean
+                        default: true
+                        hidden: true
+                    - variable: hostPath
+                      label: "Path to Z-Wave device on host"
+                      schema:
+                        type: path
+                        required: true
+                        default: "/dev/ttyUSB0"
+
+        # Zigbee device -----------------
+        - variable: zigbee
+          label: "USB Zigbee Device"
+          schema:
+            type: dict
+            attrs:
+              - variable: enabled
+                label: "USB Zigbee Device"
+                schema:
+                  type: boolean
+                  default: false
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: setPermissions
+                      label: "Automatic Permissions"
+                      description: "Automatically set permissions on install"
+                      schema:
+                        type: boolean
+                        default: false
+                        hidden: true
+                    - variable: name
+                      label: "Mountpoint Name"
+                      schema:
+                        type: string
+                        default: "ZWave"
+                        required: true
+                        editable: true
+                    - variable: emptyDir
+                      label: "emptyDir"
+                      schema:
+                        type: boolean
+                        default: false
+                        hidden: true
+                        editable: false
+                    - variable: mountPath
+                      label: "Path to mount device inside the container"
+                      description: "Preferably the same as the path to the device path on the host"
+                      schema:
+                        type: path
+                        required: true
+                        default: "/dev/ttyUSB0"
+                        editable: true
+                    - variable: hostPathEnabled
+                      label: "host Path Enabled"
+                      schema:
+                        type: boolean
+                        default: true
+                        hidden: true
+                    - variable: hostPath
+                      label: "Path to Zigbee device on host"
+                      schema:
+                        type: path
+                        required: true
+                        default: "/dev/ttyUSB0"
```

Comment:
The primary reason for this suggestion is to resolve the following error (Only the Z-Wave error is shown but the same error is repeated with Zigbee) - The suggestion fixes the error for both Z-Wave and Zigbee

```
Error: [EFAULT] Failed to install chart release: Error: failed pre-install: warning: Hook pre-install zwavejs2mqtt/templates/common.yaml failed: Job.batch "ztest-zwavejs2mqtt-config" is invalid: [spec.template.spec.volumes[1].name: Invalid value: "ZWave": a lowercase RFC 1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name', or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?'), spec.template.spec.containers[0].volumeMounts[1].name: Not found: "ZWave"]
```

Additionally, most of the variables in these sections are already hidden, this further clean-ups the Z-Wave and Zigbee device sections by removing un-needed lines.

```suggestion
        # Z-Wave device -----------------
        - variable: zwave
          label: "USB Z-Wave Device"
          schema:
            type: dict
            attrs:
              - variable: enabled
                label: "Enable Z-Wave Device"
                schema:
                  type: boolean
                  default: false
                  show_subquestions_if: true
                  subquestions:
                    - variable: hostPathEnabled
                      label: "Enable Host Path"
                      schema:
                        type: boolean
                        default: true
                        hidden: true
                    - variable: setPermissions
                      label: "Automatic Permissions"
                      description: "Automatically set permissions on install"
                      schema:
                        type: boolean
                        default: false
                        hidden: true
                    - variable: emptyDir
                      label: "Empty Dir"
                      schema:
                        type: boolean
                        default: false
                        hidden: true
                    - variable: hostPath
                      label: "Path to Z-Wave device on host"
                      schema:
                        type: path
                        required: true
                        default: "/dev/ttyACM0"
                    - variable: mountPath
                      label: "Path to mount Z-Wave device inside the container"
                      description: "Preferably the same as the path to the device path on the host"
                      schema:
                        type: path
                        required: true
                        default: "/dev/ttyACM0"
        # Zigbee device -----------------
        - variable: zigbee
          label: "USB Zigbee Device"
          schema:
            type: dict
            attrs:
              - variable: enabled
                label: "Enable Zigbee Device"
                schema:
                  type: boolean
                  default: false
                  show_subquestions_if: true
                  subquestions:
                    - variable: hostPathEnabled
                      label: "Enable Host Path"
                      schema:
                        type: boolean
                        default: true
                        hidden: true
                    - variable: setPermissions
                      label: "Automatic Permissions"
                      description: "Automatically set permissions on install"
                      schema:
                        type: boolean
                        default: false
                        hidden: true
                    - variable: emptyDir
                      label: "Empty Dir"
                      schema:
                        type: boolean
                        default: false
                        hidden: true
                    - variable: hostPath
                      label: "Path to Zigbee device on host"
                      schema:
                        type: path
                        required: true
                        default: "/dev/ttyACM1"
                    - variable: mountPath
                      label: "Path to mount Zigbee device inside the container"
                      description: "Preferably the same as the path to the device path on the host"
                      schema:
                        type: path
                        required: true
                        default: "/dev/ttyACM1"
```

Reply:
Is this in line with the standards set out in the wiki?

Reply:
I'll merge it, but I still need the examples added to the documentation about devicemounts in /docs...